gradle/libs.versions.toml

```diff
@@ -7,12 +7,21 @@ jline = "3.22.0"
 network = { group = "com.nukkitx.network", name = "raknet", version = "1.6.28-SNAPSHOT" }
 natives = { group = "com.nukkitx", name = "natives", version = "1.0.3" }
 fastutil = { group = "com.nukkitx", name = "fastutil-lite", version = "8.1.1" }
+fastutil1 = { group = "com.nukkitx.fastutil", name = "fastutil-int-short-maps", version = "8.5.3" }
```

Comment:
Could we target the cloudburst snapshot versions here? I would rather not having people using this version as it has some issue with the source jars containing the entire fastutil distribution.